src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -762,6 +823,11 @@
                (i18n/trs ":localcacert ({0}) could not be found and no file at :cacert ({1}) to copy it from"
                     localcacert cacert))))))
 
+;; Open/question - Is this the method that would be invoked when an agent requests
+;; for a CRL over HTTP/REST? If so once we have the infra-crl functioning then perhaps
+;; this would need to be updated 
```

Comment:
No, what this does is copy the cacrl from its location to the location that the agent/master will use it (localcacrl). Its used on start up, but also we register a file watcher for the cacrl, so when it is updated the master will copy the file over and refresh its ssl-context.